### PR TITLE
35 handle session

### PR DESCRIPTION
**Pull request by bartorl**
Added session menagment and reading _id from session

### CODE REVIEW

File: controllers/authController.js [hunk 1]
```diff
@@ -41,6 +41,8 @@ const login_post = async (req, res) => {
             const applicant = await Applicant.login(email, password);
             if (applicant) {
                 console.log("Loged in");
+                sessionController.sessionAuthentication(req);
+                sessionController.sendAplicantInfoToSession(req, applicant.name, applicant.email, applicant._id);
```
Comment (macibol775) [inline controllers/authController.js:45]: aPPlicant

File: controllers/sessionController.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const sessionAuthentication = (req) => {
```
Comment (MasterCoookie) [inline controllers/sessionController.js:1]: Name method as method

File: controllers/sessionController.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    req.session.authenticated = true;
+};
+
+const sendAplicantInfoToSession = (req, name, email, _id) => {
```
Comment (macibol775) [inline controllers/sessionController.js:5]: aPPlicant

File: controllers/sessionController.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    req.session.applicant = { name: name, _id: _id, email: email };    
+};
+
+const destroySession = (req) => {
+    req.session.destroy();
+};
+
+module.exports = {
+  sessionAuthentication,
+  sendAplicantInfoToSession,
```
Comment (macibol775) [inline controllers/sessionController.js:15]: aPPlicant

File: controllers/authController.js [hunk 5]
```diff
@@ -41,6 +41,8 @@ const login_post = async (req, res) => {
             const applicant = await Applicant.login(email, password);
             if (applicant) {
                 console.log("Loged in");
```
Comment (tomazaw) [inline controllers/authController.js:43]: Log and typo inside log

File: controllers/sessionController.js (hunk 3 above)
Comment (MasterCoookie) [inline controllers/sessionController.js:5]: We dont send anythong, we save/store it

File: controllers/authController.js (hunk 1 above)
Comment (macibol775) [reply controllers/authController.js:45]: name + surname

**Review by MasterCoookie — CHANGES_REQUESTED** [review]
Rename pls

**Review by macibol775 — CHANGES_REQUESTED** [review]
typos

**Review by macibol775 — APPROVED** [review]

**Review by MasterCoookie — APPROVED** [review]

**Review by WojciechPtas — APPROVED** [review]
LGTM

**Pull request merged**